recipes-external/external-toolchain/external-toolchain.bb

```diff
@@ -50,6 +50,20 @@ do_install () {
     cp -a "${EXTERNAL_TOOLCHAIN}/." "${D}/${SDKPATHTOOLCHAIN}/"
 }
 
+LICENSE_PATH += "${WORKDIR}/licenses"
+
+link_license () {
+    if ! [ -e "${LICENSE_FILE}" ]; then
+        bbwarn "No license file present at ${LICENSE_FILE}, skipping"
```

Comment:
So when a toolchain will not have the license file, it will give this warning. Am I correct?

Reply:
@kergoth I get this warning
 external-toolchain-riscv64-1.0-r0 do_populate_lic: No license file present at /home/ahsann/siemens/embedded_riscv_13.0.0/toolchains/riscv64-oe-linux.13.0/legal/ReadMe_OSS.html, ...
I think the message needs to be updated. It says No License file preset at and then we are mentioning file instead of location

Reply:
What do you mean by "file instead of location"?

Reply:
What do you mean by "file instead of location"?
[**@Noor-Ahsan**](https://github.com/Noor-Ahsan) commented on [Jan 16, 2023, 7:14 AM MST](https://github.com/MentorEmbedded/meta-sourcery/pull/243#discussion_r1071293133 "2023-01-16T14:14:47Z - Replied by Github Reply Comments"):
> So when a toolchain will not have the license file, it will give this warning. Am I correct?

Correct.

Reply:
What do you mean by "file instead of location"?
[**@Noor-Ahsan**](https://github.com/Noor-Ahsan) commented on [Jan 16, 2023, 7:14 AM MST](https://github.com/MentorEmbedded/meta-sourcery/pull/243#discussion_r1071293133 "2023-01-16T14:14:47Z - Replied by Github Reply Comments"):
> So when a toolchain will not have the license file, it will give this warning. Am I correct?

Correct.

Reply:
What do you mean by "file instead of location"?
[**@Noor-Ahsan**](https://github.com/Noor-Ahsan) commented on [Jan 16, 2023, 7:14 AM MST](https://github.com/MentorEmbedded/meta-sourcery/pull/243#discussion_r1071293133 "2023-01-16T14:14:47Z - Replied by Github Reply Comments"):
> So when a toolchain will not have the license file, it will give this warning. Am I correct?

Correct.

Reply:
@kergoth  shouldn't we drop  ReadMe_OSS.html from 

No license file present at /home/ahsann/siemens/embedded_riscv_13.0.0/toolchains/riscv64-oe-linux.13.0/legal/ReadMe_OSS.html,
?

Reply:
I don't feel like it matters too much. If it says `present in` then it would have to be the directory to make sense, but `at` feels like it can refer to the precise path or the directory it should reside in. But the user won't know the expected filename, only directory, if the message doesn't specify it, as it only says "license file", not "ReadMe_OSS.html", but it only will use the latter.